### ISSUE TITLE
Select text with keyboard

### ISSUE THREAD

**Issue by eueiwojfudn** [giuspen/cherrytree#1441]
I can't select a word or word(s) in text mode with the keyboard.

Let's say, as example, I would like with "shift + arrow" to select a word at the cursor position in order make it "bold or italic" afterwards.

This is something I can't do with the keyboard. Maybe it's a bug.

I would have to select the word or word(s) with the mouse instead.

---------------------

And... it's matter of preference I may say, but the "save icon" looks to me more as a "download icon" than a "save icon". 
But I can live with that with no problem at all. 

Thank you for your excellent work !

**Comment by giuspen** [comment]
@userfav probably @eueiwojfudn is right about the "save icon" looking more as a "download icon", could you help with this? (no rush).
About the text selection with keyboard it is fixed now, thanks for reporting, will be ok in 0.99.27.

**Comment by userfav** [comment]
@giuspen Ok, I will take a look, maybe an good ol' floopy disk.

**Comment by Crankle437** [comment]
Perhaps the issues is fixed in 0.99.27, but on Ubuntu 20.04 cherrytree crashes sometimes by select text with the error keys in the current version 0.99.26. The error shows that there are some deprecated functions in use, I'm not sure if this correspond to cherrytree:

```
XsessionErrors:
 (xfce4-session:1508): GLib-WARNING **: 13:51:07.072: GChildWatchSource: Exit status of a child process was requested but ECHILD was received by waitpid(). See the documentation of g_child_watch_source_new() for possible causes.
 (wrapper-2.0:2194): Gtk-WARNING **: 13:51:08.234: Negative content width -3 (allocation 1, extents 2x2) while allocating gadget (node button, owner GtkToggleButton)
 (wrapper-2.0:2204): Gtk-WARNING **: 13:51:08.284: Negative content width -3 (allocation 1, extents 2x2) while allocating gadget (node button, owner PowerManagerButton)
 (xfce4-session:1508): xfce4-session-WARNING **: 13:51:08.326: Unable to launch "baloo_file" (specified by autostart/baloo_file.desktop): Failed to execute child process “baloo_file” (No such file or directory)
 (xfce4-session:1508): xfce4-session-WARNING **: 13:51:07.869: Unable to launch "/usr/lib/x86_64-linux-gnu/libexec/kdeconnectd" (specified by autostart/kdeconnectd.desktop): Failed to execute child process “/usr/lib/x86_64-linux-gnu/libexec/kdeconnectd” (No such file or directory)
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:07.872: Theme parsing error: <data>:1:46: The style property GtkWidget:focus-padding is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:07.873: Theme parsing error: <data>:1:78: The style property GtkWidget:focus-line-width is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:07.873: Theme parsing error: <data>:1:108: The style property GtkButton:default-border is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:07.873: Theme parsing error: <data>:1:136: The style property GtkButton:inner-border is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:07.899: Theme parsing error: <data>:1:46: The style property GtkWidget:focus-padding is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:07.899: Theme parsing error: <data>:1:78: The style property GtkWidget:focus-line-width is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:07.899: Theme parsing error: <data>:1:108: The style property GtkButton:default-border is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:07.899: Theme parsing error: <data>:1:136: The style property GtkButton:inner-border is deprecated and shouldn't be used anymore. It will be removed in a future version
 (polkit-gnome-authentication-agent-1:2365): GLib-CRITICAL **: 13:51:08.160: g_variant_new_string: assertion 'string != NULL' failed
 (polkit-gnome-authentication-agent-1:2365): polkit-gnome-1-WARNING **: 13:51:08.161: Failed to register client: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.gnome.SessionManager was not provided by any .service files
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:08.523: Theme parsing error: <data>:1:46: The style property GtkWidget:focus-padding is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:08.523: Theme parsing error: <data>:1:78: The style property GtkWidget:focus-line-width is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:08.523: Theme parsing error: <data>:1:108: The style property GtkButton:default-border is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:08.523: Theme parsing error: <data>:1:136: The style property GtkButton:inner-border is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:11.089: Theme parsing error: <data>:1:46: The style property GtkWidget:focus-padding is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:11.089: Theme parsing error: <data>:1:78: The style property GtkWidget:focus-line-width is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:11.089: Theme parsing error: <data>:1:108: The style property GtkButton:default-border is deprecated and shouldn't be used anymore. It will be removed in a future version
 (wrapper-2.0:2203): Gtk-WARNING **: 13:51:11.089: Theme parsing error: <data>:1:136: The style property GtkButton:inner-border is deprecated and shouldn't be used anymore. It will be removed in a future version
 (Thunar:2191): thunarx-CRITICAL **: 14:04:28.651: thunarx_menu_provider_get_folder_menu_items: assertion 'thunarx_file_info_is_directory (folder)' failed
 (Thunar:2191): thunarx-CRITICAL **: 14:04:28.652: thunarx_menu_provider_get_folder_menu_items: assertion 'thunarx_file_info_is_directory (folder)' failed
 (Thunar:2191): thunarx-CRITICAL **: 14:04:28.652: thunarx_menu_provider_get_folder_menu_items: assertion 'thunarx_file_info_is_directory (folder)' failed
 (Thunar:2191): thunarx-CRITICAL **: 14:04:28.652: thunarx_menu_provider_get_folder_menu_items: assertion 'thunarx_file_info_is_directory (folder)' failed
 (Thunar:2191): thunar-WARNING **: 14:04:28.679: Content type loading failed for 598E-7404: Error when getting information for file “/media/xyz”: No such file or directory
 (cherrytree:6584): Gtk-CRITICAL **: 15:02:12.073: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar
 (cherrytree:6584): Gtk-CRITICAL **: 15:02:12.073: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

```

**Comment by giuspen** [comment]
@Crankle437 the problem with those keyboard shortcuts is that have been stolen from the tree move nodes actions, I have fixed this already, being so annoying maybe I'll release a new version already later
